deploy/crds/metal3.io_baremetalhosts_crd.yaml

```diff
@@ -157,6 +157,18 @@ spec:
                   - sha256
                   - sha512
                   type: string
+                format:
+                  description: DiskFormat contains the format of the image (raw, qcow2,
+                    ...) Needs to be set to raw for raw images streaming
+                  enum:
+                  - raw
+                  - qcow2
+                  - qed
```

Comment:
"[It is not recommended to use QED for any new images.](https://wiki.qemu.org/Features/QED)"
Let's not make this an option.